example-code/snippets/src/backend/ExampleCode.test.ts

```diff
@@ -201,6 +201,60 @@ describe("Example Code", () => {
     // __PUBLISH_EXTRACT_END__
   });
 
+  it("CodeService", async () => {
+
+    if (false) { // this will compile but it will not run, because the root elementhas no federationGuid -- waiting for a fix
+
+      // __PUBLISH_EXTRACT_START__ CodeService.reserveInternalCodeForNewElement
+      const code = Subject.createCode(iModel, IModel.rootSubjectId, "test"); // example code
+
+      const proposedCode = CodeService.makeProposedCode({ iModel, code, props: { guid: Guid.createValue() } });
+      await iModel.codeService?.internalCodes?.writeLocker.reserveCode(proposedCode);
+
+      const elementId = Subject.insert(iModel, IModel.rootSubjectId, code.value);
+      // __PUBLISH_EXTRACT_END__
+
+      // __PUBLISH_EXTRACT_START__ CodeService.updateInternalCodeForExistinglement
+      const el = iModel.elements.getElement(elementId);
+      el.code = new Code({ ...el.code.toJSON(), value: "changed" });
+      await iModel.codeService?.internalCodes?.writeLocker.updateCode({ guid: el.federationGuid!, value: el.code.value });
```

Comment:
same here with note that this will fail if "changed" is already used.

Reply:
done